src/config/env.ts

```diff
@@ -63,11 +43,26 @@ export const envVars: EnvVars = new Proxy<object>({}, {
 	}
 }) as EnvVars;
 
-// Check to see if all required environment variables are set
-const missingVars = requiredEnvVars.filter(key => envVars[key] === undefined);
-if (missingVars.length) {
-	throw new Error(`Missing required Environment Variables: ${missingVars.join(", ")}`);
-}
+envCheck(
+	"APP_ID",
+	"APP_URL",
+	"INSTANCE_NAME",
```

Comment:
`INSTANCE_NAME` is new in the check compare to existing code. Just to confirm this is intended correct?

Reply:
Yes, it's needed or else stuff fails, was just something that was overlooked in the past.